force-app/main/default/tests/Safely_Tests.cls

```diff
@@ -386,4 +386,16 @@ private class Safely_Tests {
             );
         }
     }
+
+	@IsTest
+	static void emptyCollections(){
+
+		Test.startTest();
+		new Safely().doInsert(new List<Account>());
+		new Safely().doUpdate(new List<Account>());
+		new Safely().doUpsert(new List<Account>());
+		new Safely().doDelete(new List<Account>());
+		Test.stopTest();
+
```

Comment:
Say it with me now. A unit test without assert calls is just a waste of (cpu) time.

Reply:
I was tempted to say `System.assert(true);` just to be stubborn :)